src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days >= 7) {
+    return (days * 40) - 50;
```

Comment:
```suggestion
    return rentalSum - 50;
```
consider to use variables